### ISSUE TITLE
Dawn of War: Dark Crusade, game not starting, nothing is logged by VK9

### ISSUE THREAD

**Issue by alex3kov** [disks86/VK9#66]
Arch Linux, WINE 3.17, VK9 0.28.0, win32 wineprefix.
Did the VK9 installation according to [Wiki page](https://github.com/disks86/VK9/wiki/Installation-Instructions-(Linux)) (can't find any *.spv files anywhere though, are they important?)
Game crashes on start (doesn't get to any graphics at all). VK9.log file remains empty. I tried setting EnableDebugLayers to 1 and LogLevel to 1 and 5 in VK9.conf - still nothing.

**Comment by disks86** [comment]
The instance error looks like enabling validation layers without the sdk installed. Can you try installing that to see if you get useful output?

**Comment by alex3kov** [comment]
Sorry, can you clarify please - install which SDK and where? I have all vulkan-devel stuff installed on the host OS.

**Comment by alex3kov** [comment]
Also Dota 2 runs fine on Vulkan on my system.

**Comment by disks86** [comment]
I was talking about the Vulkan SDK. The reason I mentioned that is "vkCreateDebugReportCallbackEXT" is only used if the dialog layers are enabled from the VK9 config. If the debug layers are enabled you'll need the SDK though. Regardless it was unable to create a Vulkan instance which leads me to believe there is something up with Vulkan in Wine on your machine. When you ran vulkaninfo was that in wine?

**Comment by alex3kov** [comment]
OK, thanks, I've set debug to 0, VK9.conf now looks like this:
```
LogFile = /tmp/VK9.log
LogLevel = 3
EnableDebugLayers = 0
```
And I'm getting a different error now, here's the WINE log: https://gist.github.com/alex3kov/0571bc1b8bbe800d62a01f7a16a59be2  
![2018-10-06-102812_224x82_scrot](https://user-images.githubusercontent.com/3616242/46568746-e28a0f00-c952-11e8-872f-f42c1940af92.png)
Still nothing in VK9.log btw.

**Comment by qwertychouskie** [comment]
Probably you are passing a Linux path as the log path although VK9 is running in Wine.  Try `Z:\tmp\VK9.log` for the path (the `Z` drive in Wine maps to `/`).

**Comment by alex3kov** [comment]
Indeed!  Here's VK9 log: https://gist.github.com/alex3kov/be387a1df122701ed25c7608ede498f9
